### DISCUSSION TITLE
support exemplar in alert

### DISCUSSION THREAD

**Discussion by bigfish49** [grafana/grafana#56917] [Alerting]
include exemplars info as a template variable in alert message?

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
